src/servletTutorial/servlets/EditBookServlet.java

```diff
@@ -12,68 +12,40 @@
 import servletTutorial.manage.BookManagement;
 import servletTutorial.validation.BookValidation;
 
-/**
- * Servlet implementation class EditBookServlet
- */
 @WebServlet("/EditBookServlet")
 public class EditBookServlet extends HttpServlet {
 	private static final long serialVersionUID = 1L;
 
-	/**
-	 * @see HttpServlet#HttpServlet()
-	 */
 	public EditBookServlet() {
 		super();
-		// TODO Auto-generated constructor stub
 	}
 
-	/**
-	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse
-	 *      response)
-	 */
 	protected void doGet(HttpServletRequest request, HttpServletResponse response)
 			throws ServletException, IOException {
-		// TODO Auto-generated method stub
-		int bookId = Integer.parseInt(request.getParameter("id"));
 
-		request.setAttribute("data", BookManagement.showEditBook(bookId));
+		int bookId = Integer.parseInt(request.getParameter("id"));
+		request.setAttribute("data", BookManagement.getBookById(bookId));
 		RequestDispatcher dispatcher = this.getServletContext()
 				.getRequestDispatcher("/WEB-INF/view/addAndEditBook/index.jsp");
 		dispatcher.forward(request, response);
-
 	}
 
-	/**
-	 * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse
-	 *      response)
-	 */
 	protected void doPost(HttpServletRequest request, HttpServletResponse response)
 			throws ServletException, IOException {
-		// TODO Auto-generated method stub
+
 		int bookId = Integer.parseInt(request.getParameter("id"));
 		String bookName = request.getParameter("name");
 		String author = request.getParameter("author");
+		String errors = BookValidation._validationBookName(bookId, bookName);
```

Comment:
The method name should be BookValidator.validate(bookName), and in that method, you have a _checkNull method. It should return a list of errors. If the list is empty, mean no errors. It helps you add more validation methods later easier.